ui/browser/actions/external.js

```diff
@@ -11,7 +11,9 @@ specific language governing permissions and limitations under the License.
 */
 
 import browserDB from '../../shared/browser-db';
-import { createTab, setPageDetails, duplicateTab, closeTab } from './main-actions';
+import {
+  createTab, setPageDetails, duplicateTab, closeTab, navigatePageTo,
+} from './main-actions';
```

Comment:
We really need to stop abusing destructuring when importing or handling `props`. Not a problem here, big problem in other places.